bootstrap.d/x11-libs.yml

```diff
@@ -218,6 +218,7 @@ packages:
         - '-Dradeon=false'
         - '-Damdgpu=false'
         - '-Dnouveau=false'
+        - '-Dinstall-test-programs=true'
```

Comment:
Why is this change needed? Also, if this is actually something we need, don't forget to bump the revision, otherwise the ci won't pick it up.